app/controllers/Application.java

```diff
@@ -165,7 +213,8 @@ public static Result search() {
           .where()
           .idEq(appId).findUnique();
       return ok(searchPage.render(null, jobDetails.render(result)));
-    } else if (flowExecId != null && !flowExecId.isEmpty()) {
+    } else if (Utils.isSet(flowExecId)) {
+      IdUrlPair flowExecPair = bestSchedulerInfoMatchGivenPartialId(flowExecId, AppResult.TABLE.FLOW_EXEC_ID);
```

Comment:
I would like a config to turn this feature off in case we run into performance issues.